modules/ROOT/pages/authentication-authorization/sso-integration.adoc

```diff
@@ -141,7 +141,8 @@ First, you configure Neo4j to use OpenID Connect as an authentication and author
 The default value for `dbms.security.auth_enabled` is `true`.
 . Uncomment the settings `dbms.security.authentication_providers` and `dbms.security.authorization_providers` and change their value to `oidc-<provider>`, where `<provider>` maps to the provider name used in the configuration settings.
 This way, the OIDC connector is used as a security provider for both authentication and authorization.
-These configuration values are comma-separated lists, so if you wish to continue to use native authentication and authorization alongside SSO, then these providers can be added to the existing `native` provider:
+These configuration values are comma-separated lists, so if you wish to continue to use native authentication and authorization alongside SSO, then these providers can be added to the existing `native` provider.
+The providers will be queried in the order they are declared.
 +
 .Configuration
```

Comment:
```suggestion
.Configure Neo4j to use two OpenID Connect and the native authentication and authorization providers.
```

Reply:
I'll remove the sentence, "This example has two OpenID Connect providers configured, as well as Neo4j native authorization and authentication."